ext/storage_sources/s3_store/s3_connection.cpp

```diff
@@ -134,12 +134,14 @@ S3Connection::GetObject(const std::string &objectKey, const std::string &path) c
 
 /*
  * ObjectExists --
- *     Checks whether an object with the given key exists in the S3 bucket.
+ *     Checks whether an object with the given key exists in the S3 bucket and also retrieves
+ * objectSize.
```

Comment:
Should we change the description say `size of the object` instead of using the variable name? I.e.
```
/*
 * ObjectExists --
 *     Checks whether an object with the given key exists in the S3 bucket and also retrieves
 *     the size of the object.
 */
```

Reply:
Changed!